keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpecTests.kt

```diff
@@ -33,8 +30,8 @@ internal class ClusterSpecTests : JUnit5Minutests {
       fixture { Fixture }
 
       mapOf(
-        "YAML" to configuredTestYamlMapper(),
-        "JSON" to configuredTestObjectMapper()
+        "YAML" to configuredTestYamlMapper().registerKeelEc2ApiModule(),
+        "JSON" to configuredTestObjectMapper().registerKeelEc2ApiModule()
```

Comment:
Would like to find a neater way of doing this but I can't roll it into those factory methods as the EC2 module is not mandatory

Reply:
You could have the plugin interface define a property similar to `supportedKind` that returns the API module to register (`supportedApi`?).